pressbooks/pressbooks-book-directory-fe
# ClickCount Fetch request

This PR address #321 

This PR add a new env variable called `VITE_CLICK_COUNT_ENDPOINT` this should has to be an endpoint url that could help users to track click count in each book.

The endpoint needs to be a valid URL, with a POST endpoint and no cors headers policy.

### How to test

1. Set the `VITE_CLICK_COUNT_ENDPOINT` in your env pointing to your endopoint
2. Check your algolia database records and see the clickCount property being increased as expected
